src/uwtools/file.py

```diff
@@ -13,13 +13,14 @@
 from uwtools.config.validator import validate_internal
 from uwtools.exceptions import UWConfigError
 from uwtools.logging import log
+from uwtools.strings import STR
 from uwtools.utils.api import str2path
-from uwtools.utils.tasks import filecopy, symlink
+from uwtools.utils.tasks import directory, filecopy, symlink
 
 
-class Stager:
+class Stager(ABC):
```

Comment:
This base class is generalized now to handle both the file and the directory cases, partially by defining abstract methods to be implemented by subclasses to deal with their specific needs.